homeassistant/components/stream/fmp4utils.py

```diff
@@ -1,63 +1,45 @@
 """Utilities to help convert mp4s to fmp4s."""
-import io
 
 
-def find_box(segment: io.BytesIO, target_type: bytes, box_start: int = 0) -> int:
+def find_box(part: bytes, target_type: bytes, box_start: int = 0) -> int:
```

Comment:
Just changed this to take `bytes` instead of `BytesIO`. Actually we use this to search the init section, so we could consider renaming the parameter form `part` to `data` or `init`.

Reply:
Yeah, we are only using this to find the `moov` box in the init, but the find_box function is more general and can actually work on/was used for finding any kind of mp4 boxes (some which may not be in the init). Maybe mp4_data or mov_data?

Reply:
Renamed to `mp4_bytes`